doc/api/cli.md

```diff
@@ -357,6 +357,107 @@ Error: Access to this API has been restricted
 }
 ```
 
+### `--allow-net-udp`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.1 - Active development
+
+When using the [Permission Model][], the process will not be able to bind or connect
```

Comment:
Why is this explicitly to `bind` and `connect`? Can't we deny UDP communication completely (using nodejs APIs of course)?

If so, I think we should reword that to: `the process will not be able to create, receive, and, send UDP packages.`